### ISSUE TITLE
Not functional under android tv box

### ISSUE THREAD

**Issue by woe66** [cralin/plugin.video.DigiOnline.ro#13]
Own two tv box devices with android. Tanix mini3 2ram with android 7.13 and kodi 17.6 and other chinese tv box 1ram, android 4.4.2 and mygica installed. Under both devices, the zip downloaded, installed and insert RDS user/passw. The addon is not functional in both devices and some channels send back this popup warning 
![15905968250821453977445522361969](https://user-images.githubusercontent.com/66027022/83046754-1ba75380-a050-11ea-8a3b-bf60d9e53ef5.jpg)

**Comment by cralin** [comment]
Hi,

I have no means to test/try on any devices (android or anything else) other
than on a RaspberryPi with Libreelec 9.2.1

Best I can offer here is one of the following options:
1) send me a test hardware so I can install and try/test the code on that
specific hardware.
That is still under the assumption that whatever android hardware you're
using does actually support all/full functionality of Kodi and is not some
stripped-down kodi.

2) Enable debug logging in the addon, restart the device so addon is
restarted, replicate the issue and send me the section of the log that was
written during the time you're replicated the issue.

A better description of how to replicate the issue would also help a lot in
order to understand the flow of events leading to the issue you're
reporting.

What version of the add-on is installed when you encounter the issue ?

**Comment by cralin** [comment]
Hi,

Please try to install Kodi 18, Leia as is brings a lot of new functionality
(the InputSream Adapter used by this add-on is one of them) as compared to
Kodi 17.

**Comment by cralin** [comment]
@woe66,
Do you have the same problem with the addon after upgrading to Kodi 18 ?

**Comment by woe66** [comment]
Hello,
I did not update the kodi to version 18. The tv box Tanix  3 Max came with producer modified kodi software, online upgradable via internal app to maximum 17.6 and I was affraid to install a newer version of kodi potential resources demanding or even not functional due to hardware configuration. Happend before on another tv box with Android 4.4.2, and now support only MyGica software to run addons.
But I had installed Romy Cupidon 1.01 build which come wit a lot of other plugins and python scripts and for now the Dexterke Digi Online plugin work ok. Before, his plugin only stream for 20 seconds and shutdown, probably a missing script installed by Romy build.
